### PR TITLE
Color Picker: Improves triggering

### LINKED ISSUE #4136: Color Picker: Should trigger on "rgb to hex" and similar
## Background
This IA should trigger on the query "rgb to hex", "rgb to hex converter", etc.

## Solution

1. Add these queries (and any similar queries) to [ColorPicker.t](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/t/ColorPicker.t)

2. Update the triggering in [ColorPicker.pm](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/ColorPicker.pm) to make the tests pass

3. Send your PR

## Get Started
- [ ] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

------
IA Page:  http://duck.co/ia/view/color_picker 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @ManrajGrover

### PR DESCRIPTION
**Pull request by ManrajGrover**
## Description of new Instant Answer, or changes
Improves triggering for Color Picker IA

## Related Issues and Discussions
Fixes #4136 

## People to notify
@moollaza 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: http://duck.co/ia/view/color_picker

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/color_picker/triggers.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+color picker
+colour picker
+colorpicker
+colourpicker
+rgb to hex
+rgb to hex converter
```
Comment (moollaza) [inline share/goodie/color_picker/triggers.txt]: I would keep out the `converter` queries, and add logic to strip any expected trailing words in the `handle`, and then ensure the remainder is empty.

That way we can handle queries like "rgb to hex converter", "rgb to hex conversion", "rgb to hex color", etc. 👍

**Comment by moollaza** [comment]
@ManrajGrover Thanks! This LGTM :+1:

File: share/goodie/color_picker/triggers.txt (hunk 1 above)
Comment (ManrajGrover) [inline share/goodie/color_picker/triggers.txt]: @moollaza Can do. I'll raise a PR for it soon.

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**